srcpkgs/python3-zipfile-deflate64/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-zipfile-deflate64'
+pkgname=python3-zipfile-deflate64
+version=0.2.0
+revision=1
+wrksrc="zipfile-deflate64-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
```

Comment:
Replace with `python3-setuptools_scm` to make sure the `egg-info` version is correct.

Reply:
Switching throws an error:
```
Traceback (most recent call last):
  File "/builddir/zipfile-deflate64-0.2.0/setup.py", line 9, in <module>
    setup(
  File "/usr/lib/python3.10/site-packages/setuptools/__init__.py", line 155, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 109, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/lib/python3.10/site-packages/setuptools/dist.py", line 458, in __init__
    _Distribution.__init__(
  File "/usr/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 293, in __init__
    self.finalize_options()
  File "/usr/lib/python3.10/site-packages/setuptools/dist.py", line 851, in finalize_options
    ep(self)
  File "/usr/lib/python3.10/site-packages/setuptools_scm/integration.py", line 104, in infer_version
    _assign_version(dist, config)
  File "/usr/lib/python3.10/site-packages/setuptools_scm/integration.py", line 51, in _assign_version
    _version_missing(config)
  File "/usr/lib/python3.10/site-packages/setuptools_scm/__init__.py", line 106, in _version_missing
    raise LookupError(
LookupError: setuptools-scm was unable to detect version for /builddir/zipfile-deflate64-0.2.0.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.

For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj
```

Reply:
Nevermind, not having this issue anymore, not sure what I changed...